packages/xl-odt-exporter/src/odt/defaultSchema/blocks.tsx

```diff
@@ -50,51 +48,43 @@ const createParagraphStyle = (
       right: "end",
       justify: "justify",
     };
-    styles["fo:text-align"] =
+    paragraphStyles["fo:text-align"] =
       alignmentMap[props.textAlignment as keyof typeof alignmentMap];
   }
 
-  if (props.backgroundColor && props.backgroundColor !== "default") {
-    const color =
-      exporter.options.colors[
-        props.backgroundColor as keyof typeof exporter.options.colors
-      ].background;
-    styleChildren.push(
-      <>
-        <LoextGraphicProperties draw:fill="solid" draw:fill-color={color} />
-        <StyleParagraphProperties fo:background-color={color} />
-      </>
-    );
-  }
+  const color =
+    props.backgroundColor && props.backgroundColor !== "default"
+      ? exporter.options.colors[
+          props.backgroundColor as keyof typeof exporter.options.colors
+        ].background
+      : undefined;
 
   if (props.textColor && props.textColor !== "default") {
     const color =
       exporter.options.colors[
         props.textColor as keyof typeof exporter.options.colors
       ].text;
-    styles["fo:color"] = color;
-  }
-
-  if (
```

Comment:
was this removed intentionally? if there are no specific styles, we don't need to register a custom style right?

Reply:
`fo:color` was moved to `textStyles`, since it should be used with `style:text-properties`

When no custom properties are applied, we're now returning the `parentStyleName`